src/Components/OATHeader/OATHeader.tsx

```diff
@@ -252,6 +232,18 @@ const OATHeader = ({ elements, dispatch, state }: OATHeaderProps) => {
         return metaDataCopy;
     };
 
+    const onFilesChange = (e) => {
+        const reader = new FileReader();
+        reader.onload = () => {
+            const files = [];
+            for (const file of uploadInputRef.current.files) {
+                files.push(file);
+            }
```

Comment:
This can probably be a `.map`?

Reply:
I think we can't because uploadInputRef.current.files is not an array. It's an object
<img width="346" alt="Screen Shot 2022-06-30 at 3 19 17 PM" src="https://user-images.githubusercontent.com/87083787/176780099-e307a092-d0e7-4747-a9a7-fc4dc9a3546f.png">

Reply:
Good point. Leave it as-is then. The alternative would be to use spread `[...uploadInputRef.current.files]` or `Array.from` but that seems less tidy than what you've already got.